### ISSUE TITLE
Swatches mechanism

### ISSUE THREAD

**Issue by panvourtsis** [Orfium/orfium-ictinus#129]
## Description 
This ticket will introduce a mechanism that will detect when to change the text color based on the background passed

## Design
The design can be found [here](https://app.abstract.com/projects/96260c1d-f169-4306-ab88-2472d8e457fd/branches/master/commits/latest/files/588a69c2-49f4-4034-830e-b9725e751d61/layers/D4714E3B-23CB-48C1-880F-F47255FB2E3E?collectionId=94da2f8d-af8f-4877-99a7-9448c8affa94&collectionLayerId=10759ca1-2cbc-4a79-bf25-db49e2b34fab)
![image](https://user-images.githubusercontent.com/6433679/101036576-bc3c7a00-3583-11eb-9115-7e6f2d3e1616.png)

## Implementation
Create a new util function that will folow the design pattern
So basically whether to show white or dark text color for components that are taking freely a color from our palette

## Testing
Test various colors that are returning the correct color
